src/main/resources/avro/sequenceAnnotations.avdl

```diff
@@ -98,9 +103,9 @@ protocol SequenceAnnotations {
     }> attributes;
 
     /**
-    Identifiers of children nodes
+    Identifier for the parent annotation, if any.
     */
-    array<string> childrenIds;
+    string parentId;
```

Comment:
I'd think it would make the common case much cleaner if we didn't allow for multiple parents. Some workarounds for this are:
- create two transcript resources instead of one; link them together or have some ID which indicates that they originated from the same source data.
- put other parent IDs into a second-class tag in the attributes field.

A few of my reasons as to why I prefer parentId over childIds:
- Traversal of the tree is cleanest via Search(byParentId). The alternative is to take the list of childIds and perform a batchGet call with those IDs (which we would have to add), or N getSeqAnnRegion calls. If we have a search by parentId, it is natural for the records to have a parentId field and the childIds field has no utility for traversal.
- If the number of children for a given annotation becomes large, you wind up with an unwieldy childIds array.